### ISSUE TITLE
UX monitoring of TE Release (v1.0 25% rollout)

### ISSUE THREAD

**Issue by evelynhilbrichdavis** [department-of-veterans-affairs/va.gov-team#92321]
## Issue Description
We need to implement UX monitoring for the upcoming release of Toxic Exposure to ensure that the user experience meets our quality standards and to identify any potential issues early. This will involve both quantitative data collection and qualitative feedback to provide a comprehensive view of user interactions and satisfaction. 

**Note:** This monitoring will run the course of the 1.0-1.1 roll-out, **but the tickets have been divided to cover shorter increments of the rollout.** Each ticket includes one qualitative data pull and a few w/ just GA4 success metrics. 

### Monitoring tools: 
1. @pacerwow GA4 to track form metrics (especially opt-in, form exit, and abandonment) 
2. @evelynhilbrichdavis Medallia survey data-- qualitative and quantitative feedback 
   - Satisfaction (Feedback survey)
    - Overall experience (Intercept survey)
    - Understandability (Intercept survey)
    - Trust (Intercept survey)
3. @evelynhilbrichdavis Contact Center data (MyVA411)

### Mpnitoring Artifacts: 
1. [TE Monitoring Document](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/research/2024-08-Toxic-Exposure/TE-Monitoring.md)
2. [Review Medallia survey data to prepare for TE launch#88714](https://app.zenhub.com/workspaces/disability-benefits-experience-team-1-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/88714)

---
## Tasks
- [ ] @pacerwow Update [TE Monitoring Document](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/research/2024-08-Toxic-Exposure/TE-Monitoring.md) at each increment and share out with #benefits-disability channel
- [ ] @pacerwow Troubleshoot any strangeness/concern in the numbers with product and eng
- [ ] @pacerwow  @evelynhilbrichdavis Escalate problems to OCTO and engineering team
- [ ] @evelynhilbrichdavis  Analyze GA4 data and user feedback from Medallia surveys/CC Team
- [ ] @evelynhilbrichdavis  Compare Medallia insights with quantitative data from GA4

## Acceptance Criteria
- [ ] Reports are complete and shared with HCD Lead, Delivery Manager, and PM
- [ ] Reports are complete and shared with OCTO and engineering team 
- [ ] Keep track of findings that could support planning for upcoming Toxic Exposure usability study.  

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by evelynhilbrichdavis** [comment]
- Updated the reporting schedule in the  [ TE Monitoring document ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/research/2024-08-Toxic-Exposure/TE-Monitoring.md)
- Thought through how new timing might impact the schedule for usability testing in October

**Comment by evelynhilbrichdavis** [comment]
# Proposal for revised usability testing timeline
Context: The TE release schedule has been extended 2.5 weeks. This impacts the timing for upcoming usability testing since the timing of the study is tied to getting UX feedback on 1.1 from Medallia/MyVA411 data. As of my last check-in w/ Shannon, we imagined sessions **starting the week of 10/14.** 

We have a couple of options on how to navigate the shifting timeline-- will walk through gantt view during Bi-weekly design sync.

## Option A
Push usability testing out until 11/11 we can get substantive UX feedback on 1.1 (data for 25-75% of 1.1 will be available 10/21). 
**Pros:**
- Learnings from 1.1 user data would help us shape the study to ensure we are evaluating the IPF alert based on known problems surfaced in Medallia/MyVA411.
**Cons**
- I'm OOO for company offsite week of 10/21 (which pushes the schedule) 
- Testing 11/11 would  put us close to Thanksgiving (harder recruiting)
- Later testing puts us farther away from developing and implementing feature improvements, rather than a fast follow post-release.

## Option B
Treat user data from 1.1 as a "nice-to-have" input and begin usability sessions around 10/28. Develop research material and then refine it by looking at 1.1 10-25% user data before starting the sessions. 
**Pros:** 
- Earlier research will allow us to make improvements faster
- Recruitment could happen while I am on my offsite, allowing us to take advantage of my OOO
**Cons:**
-We will have less insight into the 1.1 user experience. That said, we don't know how useful the survey/call center data will be yet, since we haven't done a pull.

**Comment by evelynhilbrichdavis** [comment]
@pacerwow Could you please complete the 10% update?

**Comment by evelynhilbrichdavis** [comment]
Sent out requests for survey and call center data 8/28-9/20. 
Medallia request #93382
MYVA 411 request #93379

**Comment by evelynhilbrichdavis** [comment]
Need to validate 10 and 25% today/tomorrow. Have been OOO unexpectedly but picking this work backup.

**Comment by evelynhilbrichdavis** [comment]
New Medallia data for 8/28-9/20 is in! I am processing it before moving on to the Contact Center material.

Tomorrow I will reshare the excel sheet I am working from and tag, etc, but on my first pass I'm seeing no Toxic Exposure usability concerns appear in the TE pages from the 5-25% rollout data set.

**Comment by mayacarrolluxa6** [comment]
What's next on this task?

**Comment by pacerwow** [comment]
@mayacarrolluxa6 it's an open, ongoing ticket for the TE release increments from 10-50%, so we are just awaiting the TE release incrementing from 25% to 50%. However, since we don't want tickets to stay open sprint over sprint, I am changing the title to 25% and will create separate tickets for 75% and 100% for the 1.0 release, and the 1.1 release.

**Comment by evelynhilbrichdavis** [comment]
[Closed with a post to DSVA slack. ](https://dsva.slack.com/archives/C04KW0B46N5/p1729274908177519). Worked with engineers to update success and submission metrics, so we're all set to close out ticket. @pacerwow
